satpy/etc/composites/viirs.yaml

```diff
@@ -232,7 +232,7 @@ composites:
       modifiers: [sunz_corrected, rayleigh_corrected]
     optional_prerequisites:
     - name: I02
-      modifiers: [sunz_corrected_iband, rayleigh_corrected_iband]
```

Comment:
Should rayleigh be removed from the M05 band too? I suppose that is still in the "correctable" range?